Source/WebCore/animation/CSSPropertyAnimation.cpp

```diff
@@ -3688,6 +3688,7 @@ CSSPropertyAnimationWrapperMap::CSSPropertyAnimationWrapperMap()
         case CSSPropertyWebkitTextZoom:
         case CSSPropertyAdditiveSymbols:
         case CSSPropertyAlignmentBaseline:
+        case CSSPropertyAlignTracks:
```

Comment:
Please file a bug to add animation support for this.

Reply:
Done https://bugs.webkit.org/show_bug.cgi?id=248886